README.md

```diff
@@ -131,5 +152,7 @@ role, goal, and reason for each.
 Finally, link to your repo on github in the space below.
 
 Your Answer:
-```text
+```
+https://github.com/JohnWPark/project1/blob/master/readme_project.md
+
```

Comment:
I like your first 3 stories! I would say "as a user, i should see fun, colorful graphics on the cards so they can enjoy the experience more." something like that. The role always needs to be a user of your website.